code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -34,7 +34,13 @@
 		"Parrot" = "parrot",
 		"Rabbit" = "rabbit",
 		"Bear" = "bear",  //VOREStation Edit
-		"Raccoon" = "raccoon" //Chompstation Edit
+		"Raccoon" = "raccoon", //Chompstation Edit

```

Comment:
Add a comma to the end of edit comments inside of a list. Travis usually screeches if there aren't any. Actually, since this is a main file, and we have a _vr file already, maybe move it there.

Reply:
a global list can't be re-initialized in another file, unfortunately. I'd have to rig up a complicated system to add our custom content to these lists during the pregame initialization

`code\modules\mob\living\silicon\pai\pai_vr.dm:8:error: possible_chassis: re-initialization of global var`

`code\modules\mob\living\silicon\pai\pai_vr.dm:22:error: possible_say_verbs: re-initialization of global var`